lib/blockchain/install.js

```diff
@@ -117,5 +121,8 @@ function run () {
   })
 
   inquirer.prompt(questions)
-    .then(answers => processCryptos(answers.crypto))
+    .then(answers => {
+      answers.crypto.push(coinUtils.getCryptoCurrency(ETH_CRYPTO_CODE).code)
```

Comment:
Why not just use ``ETH`` instead of ``coinUtils.getCryptoCurrency(ETH_CRYPTO_CODE).code``?

Reply:
We could hard code the crypto code here (according to coin-utils.js, the code for ETH is 'code: 'ethereum').
However, I don't think it is a good practice to hard code it: 
- We have a nice set of utilities on coin-utils module,. So, every time we need to know/change some of we limit our scope of action to this specific module.
Imagine if we hard code this string among several modules and, for some reason, we need to change the string?
We would have to scan all the modules for all the hard coded strings.